src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -133,7 +133,101 @@ const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log)
       '"logging.timezone" has been deprecated and will be removed ' +
         'in 8.0. To set the timezone moving forward, please add a timezone date modifier to the log pattern ' +
         'in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md'
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
+    );
+  }
+  return settings;
+};
+
+const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.dest')) {
+    log(
+      '"logging.dest" has been deprecated and will be removed ' +
+        'in 8.0. To set the destination moving forward, you can use the "console" appender ' +
+        'in your logging configuration or define a custom one. For more details, see ' +
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const quietLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.quiet')) {
+    log(
+      '"logging.quiet" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:error" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const silentLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.silent')) {
+    log(
+      '"logging.silent" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:off" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const verboseLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.verbose')) {
+    log(
+      '"logging.verbose" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:all" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const jsonLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.json') && settings.env !== 'development') {
```

Comment:
Can we add a comment that explains why this is necessary to ignore in development?

Reply:
Do we need to ignore it in development? I know that legacy `logging.json` doesn't have an effect in dev, but it might still be useful to see a notice that it is deprecated in case you are trying to use it?

Reply:
@lukeelmers It's going to become annoying to carry on seeing these warnings and not be able to turn them off in dev mode without turning off all logs. 

We turn the warnings off in development mode because our dev CLI code actually manually specifies `--logging.json=false` [here](https://github.com/elastic/kibana/blob/master/src/dev/cli_dev_mode/using_server_process.ts#L28) because it's executed in a child process and [this logic here](https://github.com/elastic/kibana/blob/master/packages/kbn-legacy-logging/src/schema.ts#L58) about the TTY check would return true if it wasn't specified.

Reply:
@joshdover I added a comment to explain why we ignore the deprecation in dev mode